File: VB/DXWebApplication1/Controllers/QueryBuilderController.vb
Imports DevExpress.Web.Mvc.Controllers
Imports System.Web.Mvc

Namespace DXWebApplication1.Controllers
    Public Class QueryBuilderController
        Inherits QueryBuilderApiControllerBase

        '
        ' GET: /QueryBuilder/

        Public Overrides Function Invoke() As ActionResult
            Dim result = MyBase.Invoke()
            ' Allow cross-domain requests.
            Response.AppendHeader("Access-Control-Allow-Origin", "*")
            Return result
        End Function

    End Class
End Namespace

File: CS/DXWebApplication1/Controllers/QueryBuilderController.cs
using DevExpress.Web.Mvc.Controllers;
using System.Web.Mvc;

namespace DXWebApplication1.Controllers
{
    public class QueryBuilderController : QueryBuilderApiControllerBase
    {
        //
        // GET: /QueryBuilder/

        public override ActionResult Invoke()
        {
            var result = base.Invoke();
            // Allow cross-domain requests.
            Response.AppendHeader("Access-Control-Allow-Origin", "*");
            return result;
        }

    }
}